### PR TITLE
Additional events

### PR DESCRIPTION
**Pull request by 0xNuggan**
This adds all necessary events and tests for their emission

### CODE REVIEW

File: src/modules/fundingManager/IFundingManager.sol [hunk 1]
```diff
@@ -21,25 +21,25 @@ interface IFundingManager {
     // Events
 
     /// @notice Event emitted when a deposit takes place.
-    /// @param from The address depositing tokens.
-    /// @param to The address that will receive the receipt tokens.
-    /// @param amount The amount of tokens deposited.
+    /// @param _from The address depositing tokens.
```
Comment (FHieser) [inline src/modules/fundingManager/IFundingManager.sol:24]: This is the only place you put an _ before the parameters. Why?

File: src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol [hunk 2]
```diff
@@ -206,8 +206,11 @@ abstract contract RedeemingBondingCurveFundingManagerBase is
         if (_fee > BPS) {
             revert RedeemingBondingCurveFundingManager__InvalidFeePercentage();
         }
-        emit SellFeeUpdated(_fee, sellFee);
+        uint oldSellFee = sellFee;
```
Comment (FHieser) [inline src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol]: Why did you change that?

File: src/modules/paymentProcessor/StreamingPaymentProcessor.sol [hunk 3]
```diff
@@ -518,6 +518,8 @@ contract StreamingPaymentProcessor is Module, IStreamingPaymentProcessor {
             .length - 1];
 
         activeVestingWallets[client][paymentReceiver].pop();
+
+        //TODO check if event is missing?
```
Comment (FHieser) [inline src/modules/paymentProcessor/StreamingPaymentProcessor.sol]: TODO

**Review by FHieser — APPROVED** [review]
LGTM
I didnt find anything mayor.

File: src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol (hunk 2 above)
Comment (0xNuggan) [reply src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol]: The event was misplaced on my branch and returning the same number twice. I agree emitting before is the more elegant solution. fixed

File: src/modules/fundingManager/IFundingManager.sol (hunk 1 above)
Comment (0xNuggan) [reply src/modules/fundingManager/IFundingManager.sol:24]: because "for" is a protected keyword, but more intuitive than "to"

File: src/modules/paymentProcessor/StreamingPaymentProcessor.sol (hunk 3 above)
Comment (0xNuggan) [reply src/modules/paymentProcessor/StreamingPaymentProcessor.sol]: Fixed. The event gets emitted in _afterClaimCleanup()
